x/ibc/spec/01_concepts.md

```diff
@@ -6,3 +6,20 @@ order: 1
 
 > NOTE: if you are not familiar with the IBC terminology and concepts, please read
 this [document](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_TERMINOLOGY.md) as prerequisite reading.
+
+### Connection Version Negotation
+
+During the handshake procedure for connections a version string is agreed upon between the two parties.
+This occurs during the first 3 steps of the handshake.
+
+During OpenInit, party A is expected to set all the versions they wish to support within their connection state.
```

Comment:
Document that order of versions is most preferred to least preferred

Reply:
I added, though this is not necessary for party A to do in OpenInit in regards to the SDK's implementation of IBC. OpenTry makes zero assumptions on the ordering of the counterparty's versions.

Reply:
let me know if you agree with the updated spec or not